docs/HOW_TO_OVERRIDE_STYLES_PROPERLY.md

```diff
@@ -0,0 +1,144 @@
+# How to Override Styles Properly
+
+### **tl;dr**
+1. Override styles using `-st-mixin` and use the Stylable variables we expose.
+   Don't target internal elements!
+2. Visually test your app, especially at points where you use wix-ui-tpa
+3. Test your app/components with different site palettes, fonts and numbers.  
+   Run tests using both a light theme and a dark theme.
+
+
+### How to override a component's stylesheet?
+> Be sure to read our [Usage document](
+https://github.com/wix/wix-ui-tpa/blob/master/docs/USAGE.md#overriding-predefined-style-params)
+
+wix-ui-tpa components are wired to users' sites palettes and fonts out-of-the-box.  
+In addition, most of the components also allow TPA's to extend their style. We achieve this by 
+using Stylable's `-st-mixin`.
+
+Each component declares the variables it exposes, like this:
+```css
+/* Overrides */
+:vars {
+   /* 
+     Text Color
+     @default color-5
+   */
+   MainTextColor: --overridable;
+}
+```
+These variables will be found in the `Style API` tab in the component's storybook.
+
+If the component uses the new approach for overrides (using the `overrideStyleParams` class), 
+use that to extend the component (as described in the aforementioned [guide](
+https://github.com/wix/wix-ui-tpa/blob/master/docs/USAGE.md#an-optimized-way-for-style-overriding---new))  
+If not, then use its "default" `.root` export for overrides.
+
+In any case **DO NOT** use `-st-extends` to override styles by targeting internal elements.
+
+#### Examples
+
+✅ DO
+
+```css
+:import {
+   -st-from: "wix-ui-tpa/.../Component.st.css";
+   -st-named: overrideStyleParams;
+   
+   /* If overrideStyleParams doesn't exist use: */
+   /* -st-default: TPAComponent; */
+}
+
+.myComponent {
+   -st-mixin: overrideStyleParams(
+     TextColor <your-color-here>
+   );
+
+   /* If overrideStyleParams doesn't exist use: */
+   /* -st-mixin: TPAComponent(...); */
+}
+```
+
+❌ DON'T
+
+```css
+:import {
+   -st-from: "wix-ui-tpa/.../Component.st.css";
+   -st-default: TPAComponent; 
+}
+
+.myComponent {
+   -st-extends: TPAComponent;
+}
+
+.myComponent::innerElement {
+   background-color: <some-color>;
+}
+```
+
+## FAQ's
+
+#### "The component doesn't expose the variables I need, what should I do?"
+First, contact us at [#wix-ui-tpa](slack://channel?id=CJRU3U97A&team=T02T01M9Y).  
```

Comment:
It seems like the link doesn't work:
https://github.com/wix/wix-ui-tpa/blob/790a37f2cc5bf620714409b71c7ff3f9f9953921/docs/HOW_TO_OVERRIDE_STYLES_PROPERLY.md#the-component-doesnt-expose-the-variables-i-need-what-should-i-do

I think we need to use full with HTTP protocol (not slack) if exists